src/nrtk_explorer/app/ui/image_list.py

```diff
@@ -297,10 +329,10 @@ def __init__(self, on_scroll, on_hover, **kwargs):
                                         "Original: Detection Annotations",
                                         classes="text-center",
                                     )
-                                    ImageDetection(
+                                    ImageWithSpinner(
                                         identifier=("props.row.original",),
                                         src=("props.row.original_src",),
-                                        annotations=("props.row.originalAnnotations",),
+                                        annotations="props.row.originalAnnotations",
```

Comment:
it is kind of weird that annotations behave differently than src... I would put tuples everywhere for consistency

Reply:
✅